lib/ZumoOta/src/App.cpp

```diff
@@ -83,17 +101,13 @@ void App::setup()
     Serial.print("IP Address: ");
     Serial.println(WiFi.localIP());
 
-    server.on("/", HTTP_GET, [](AsyncWebServerRequest *request){
-        request->send(200, "text/plain", "Hello, world");
-    });
-
-    server.begin();
-
+    fileManager.init();
+    start();

```

Comment:
is this start really needed in addition to setup ? 
What is your rule to do things in setup() or in start() ? At least for me it is unclear why filemanager init() is here, but webServer.init() in start()

Reply:
I will modify the start() Method by the next commit. Start() Method will be responsible for initializing and starting the application. It will primarily involves initializing the FileManager and WebServerCustom components.